src/iris/bin/owasync.py

```diff
@@ -15,6 +15,10 @@
 import requests
 
 
+RELAY_IGNORE = 0
```

Comment:
Is there a reason for differentiating between the RELAY_IGNORE and RELAY_SUCCESS? It seems like the code only checks if it was a relay failure or not and handles a relay_success and relay_ignore in the same way, perhaps we can avoid defining global variables and just return true or false for if the message was handled successfully or not.